fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt

```diff
@@ -112,6 +104,58 @@ class NotificationRobot {
         }
     }
 
+    fun swipeDownloadNotification(
+        direction: String,
+        shouldDismissNotification: Boolean,
+        canExpandNotification: Boolean = true,
+    ) {
+        for (i in 1..6) {
```

Comment:
What is 1 and 6? Please add a comment, and function level comment on what this is doing.

Reply:
Done !